tests/test_wrappers.py

```diff
@@ -306,32 +306,47 @@ def test_compression(vfile, group_name, name, sparse):
         assert ds.compression_opts == 5
 
 
-def test_compression_hotswap_to_inmemoryarraydataset(vfile):
-    """Test .compression and .compression_opts properties after DatasetWrapper hot-swaps
+def test_filters_hotswap_to_inmemoryarraydataset(vfile):
+    """Test .compression, .compression_opts, .shuffle, and .fletcher32 after
+    DatasetWrapper hot-swaps
```

Comment:
can't test `scaleoffset` in the same test as it is mutually exclusive with `fletcher32`